hosting/technical/catalog_instructions.qmd

```diff
@@ -0,0 +1,41 @@
+# Creating Intake catalogs
+
+There are two versions of [intake](https://intake.readthedocs.io/en/latest/). Here, we focus on intake version 1. Version 2 has advanced features, but a [(partially) incompatible syntax](https://intake.readthedocs.io/en/latest/scope2.html#relationship-to-v1).
```

Comment:
This sounds like a mess? Is version 2 compatible with version 1 if we work with version 1 is it backward compaible with version 2? Sound like it is not?

Reply:
Yes, it is a mess and the transition to v2 is not really backwards compatible. We are still debating how to tackle this: the options are Intake v2 or STAC (with a tendency towards the latter). Unfortunately, both are much more cumbersome to build.

Reply:
Yes, it's a mess. I'd probably recommend to stick with either v1, or go towards STAC (with a tendency to the latter).
This probably should also depend on the variety of working environments we expect to deal with. If everyone is running Python anyways, we could stick on v1 for simplicity, but if we want to support more languages, we probably don't have that many choices except STAC.

One more reason for intake might be the ability to hack around broken datasets, however we also just might not want to do that.